src/index.html

```diff
@@ -11,7 +11,145 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+     action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+     method="post"
+     autocomplete="off"
+     >
+      <fieldset >
+        <legend>Personal information</legend>
+        <label class="form_field">
+           Surname:
+          <input
+           type="text"
+           min="2"
+           class="field">
+        </label>
+
+        <label class="form_field">
+          Name:
+          <input
+           type="text"
+           max="10"
+           class="field">
```

Comment:
```suggestion
          <input
             type="text"
             max="10"
             class="field"
           >
```
проверь правильность форматирования своего кода(смотри Good Example)
<img width="1178" alt="formatting" src="https://user-images.githubusercontent.com/94292654/178000433-e87a97ee-335a-49f7-ba9d-3690f9722c4c.png">

Reply:
Исправил. Но все равно на доли мм не сходится.